### PR TITLE
Add several spelling corrections and refinements

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3716,6 +3722,10 @@ codepoitn->codepoint
 codesc->codecs
 codespel->codespell
 coditions->conditions
+coduct->conduct
+coducted->conducted
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add `coductor->conductor` and probably `coducter->conductor`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4701,6 +4711,7 @@ constructer->constructor
 constructers->constructors
 constructes->constructs
 constructred->constructed
+constructt->construct
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Could add the s, ed, or, ors and ing variants of this typo too.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -5361,11 +5372,15 @@ croch->crotch
 crockadile->crocodile
 crockodiles->crocodiles
 cronological->chronological
+cros->cross
 croshet->crochet
+crosreference->cross-reference
+crosreferences->cross-references
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the ed version too.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -5361,11 +5372,15 @@ croch->crotch
...
 cross-commpilation->cross-compilation
 cross-orgin->cross-origin
 crossin->crossing
 crossreference->cross-reference
+crossreferences->cross-references
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the ed version too.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -14176,7 +14217,7 @@ ovverridden->overridden
 ovverride->override
 ovverrides->overrides
 ovverriding->overriding
-ower->lower
+ower->lower, owner,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest over.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 900c7b3

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 900c7b3

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 900c7b3

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 900c7b3

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 900c7b3

**Pull request merged**